src/org/javarosa/core/model/utils/DateUtils.java

```diff
@@ -270,8 +272,7 @@ public static String format (DateFields f, String format) {
 				} else if (c == 'n') {	//numeric month
 					sb.append(f.month);
 				} else if (c == 'b') {	//short text month
-					String[] months = new String[] {"Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"};
-					sb.append(months[f.month - 1]);
+                    sb.append(DateTimeFormat.forPattern("MMM").print(new DateTime(DateUtils.getDate(f))));
```

Comment:
I've just skimmed the JodaTime docs, and it refers to the "default timezone" a lot.  Does this mean the OS timezone, rather than defaulting to e.g. `UTC` for everyone?